Assignment/oddeven.txt

```diff
@@ -0,0 +1,21 @@
+/* [16] WAP to check number is Even or Odd
+ID : 23BCA261 */
+
+#include<iostream.h>
+#include<conio.h>
+void main()
+{
+	int n;
+	clrscr();
+	cout<<"Enter a number : ";
+	cin>>n;
+	if(n%2==0)
+	{
+		cout<<n<<" is Even number";
+	}
+	else
+	{
+		cout<<n<<" is Odd number";
+	}
+	getch();

```

Comment:
why dont you use return 0?

Reply:
ok its my mistake . i have to write return 0. but here i declare main function as void main() so there is no need to write return 0.